### PR TITLE
bpo-27931: Fix email address header parsing error

### PR DESCRIPTION
**Pull request by jayyyin**
https://bugs.python.org/issue27931

https://bugs.python.org/issue27931

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately our records indicate you have not signed the CLA. For legal reasons we need you to sign this before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.


Thanks again to your contribution and we look forward to looking at it!

File: Misc/NEWS.d/next/Library/2018-01-25-21-04-11.bpo-27931.e4r52t.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Email parse IndexError <""@wiarcom.com> patch made by xiang.zhang
```
Comment (bitdancer) [inline Misc/NEWS.d/next/Library/2018-01-25-21-04-11.bpo-27931.e4r52t.rst]: I suggest: Fix email address header parsing error when the username is an empty quoted string.  Patch by xian.zhang.

**Review by bitdancer — CHANGES_REQUESTED** [review]

**Comment by jayyyin** [comment]
did the change to the news file as requested

File: Misc/NEWS.d/next/Library/2018-01-25-21-04-11.bpo-27931.e4r52t.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+Fix email address header parsing error when the username is an empty quoted string. Patch by xian.zhang.
```
Comment (zhangyangyu) [inline Misc/NEWS.d/next/Library/2018-01-25-21-04-11.bpo-27931.e4r52t.rst]: Hmm, my name is Xiang Zhang.

**Comment by jayyyin** [comment]
fixed the typo in the name, sorry for not noticing the mistake

**Review by bitdancer — APPROVED** [review]

**Pull request merged**